tsl/src/bgw_policy/continuous_aggregate_api.c

```diff
@@ -384,7 +367,7 @@ interval_to_int128(const Interval *interval)
 	return span;
 }
 
-static int64
+int64
```

Comment:
Really minor, I'm just wondering:
Since `interval_to_int64` is declared in `policies_v2.h` perhaps it might be better to move the definition inside `policies_v2.c`?

Reply:
Ideally it should be in something like time utils. But I don't think we should do the refactoring in this PR at the moment.